crates/tauri-bundler/src/bundle/windows/nsis/mod.rs

```diff
@@ -517,6 +525,29 @@ fn build_nsis_app_installer(
   ));
   fs::create_dir_all(nsis_installer_path.parent().unwrap())?;
 
+  #[cfg(target_os = "windows")]
+  if settings.can_sign() {
+    log::info!("Signing NSIS plugins");
+    for dll in glob::glob(
+      _nsis_toolset_path
+        .join("Plugins/*/*.dll")
+        .to_string_lossy()
+        .to_string()
+        .as_str(),
+    )? {
+      let path = dll?;
+      if NSIS_PLUGIN_DLLS.contains(
+        &path
+          .file_name()
+          .unwrap_or_default()
+          .to_string_lossy()
+          .as_ref(),
+      ) {
+        try_sign(&path, settings)?;
+      }
+    }
```

Comment:
we know the paths of these dlls, no need to glob
```suggestion
    for dll in NSIS_PLUGIN_DLLS {
      let path = _nsis_toolset_path.join("Plugins/x86-unicode").join(dll);
      try_sign(&path, settings)?;
    }
```